werkit/orchestrator/orchestrator_lambda/parallel.py

```diff
@@ -6,7 +6,7 @@
 from harrison import Timer
 
 
-lambda_client = boto3.client("lambda")
+# lambda_client = boto3.client("lambda")
```

Comment:
Can you revert this (or fix it)?

Reply:
Fixed in e89c1cf2a3361a00c8d64177ecbb29b8b8555ff9